### DISCUSSION TITLE
Migrate from nextcloudpi

### DISCUSSION THREAD

**Discussion by baschior** [nextcloud/all-in-one#755] [Questions] [answered]
Can anyone give me a hint how complicate a Migration from ncp would be and what the benefits of nc aio are compared to ncp?

**Comment by szaimen** [comment, marked as answer]
Migration is pointed out here: https://github.com/nextcloud/all-in-one/blob/main/migration.md

Regarding the benefits, I guess being able to update it independently from the host OS and other applications and the included optional addons are worth to be mentioned here...

**Reply by baschior** [reply]
Puh Migration of folders only is not an Option for me... There are 12 users with calendars and stuff...

**Reply by szaimen** [reply]
Yes, database migration is also explained there...
